packages/paste-core/components/typography/src/Heading.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+
+export type asTags = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'div' | 'label' | 'span';
+export type HeadingStyle =
+  | 'headingStyle10'
+  | 'headingStyle20'
+  | 'headingStyle30'
+  | 'headingStyle40'
+  | 'headingStyle50'
+  | 'headingStyle60';
+
+interface Heading {
+  as: asTags;
+  id?: never;
+  className?: never;
+  children: NonNullable<React.ReactNode>;
+  headingStyle?: HeadingStyle;
```

Comment:
Should we call this `variant` so it's consistent with other components?

Reply:
Oooooh I dig it